static/app/components/modals/sudoModal.tsx

```diff
@@ -35,12 +35,37 @@ type Props = WithRouterProps &
 type State = {
   busy: boolean;
   error: boolean;
+  errorType: string;
+  isFirstStep: boolean;
+  superuserAccessCategory: string;
+  superuserReason: string;
 };
 
 class SudoModal extends React.Component<Props, State> {
   state: State = {
     error: false,
+    errorType: '',
     busy: false,
+    isFirstStep: true,
```

Comment:
nit: 

fwiw, if you're counting steps, i'd typically prefer to just have `stepCount: number` rather than `isFirstStep`.
with the current implementation it seems fine - but then if there's ever any more steps added it gets kinda weird.
Or if someone misreads something and iterates the step accidentally or soemthing 🤷 
dunno not a huge deal lol